lightning/src/ln/chan_utils.rs

```diff
@@ -497,8 +497,8 @@ pub fn build_htlc_transaction(prev_hash: &Txid, feerate_per_kw: u32, to_self_del
 
 #[derive(Clone)]
 /// We use this to track local commitment transactions and put off signing them until we are ready
-/// to broadcast. Eventually this will require a signer which is possibly external, but for now we
-/// just pass in the SecretKeys required.
+/// to broadcast. This class can be used inside a signer implementation to generate a signature
+/// given the relevant secret key.
```

Comment:
Seems to be getting rid of some info here -- rather than losing it, maybe add a sentence at the end: "Eventually, a signer (whether external or internal) will be required, but for now the SecretKeys are passed in directly when need be."

Reply:
`LocalCommitmentTransaction.get_local_sig` is now only called inside `ChannelKeys` implementations, so I believe the new wording is accurate.  I think the original wording is a holdover from the time that the secret key was passed out of the control of `ChannelKeys` implementations, but now the encapsulation of secret keys is complete due to the recent changes in that area by @ariard.